### PR TITLE
[WIP]: Improve compare css values

### PR DESCRIPTION
**Pull request by darlanmendonca**
to fix errors with different order in css, example

```css
// implemented
box-shadow: 0 0 0 red;
```

```css
// computed
box-shadow: rgb(255, 0, 0) 0px 0px 0px;
```

note, the box-shadow accept the following arguments`h-shadow v-shadow blur spread color`. So in the "implemented" css, I dont define spread

now in "computed" css, the browser return in different order, color goes to the first argument, browser convert units, to a pixel, and set the missing arguments (spread) to 0px

this PR try resolve this, creating a "fake" css.

basically, I create a iframe, add a fake element, set the style expected, compute their, and compare with received in element.

### CODE REVIEW

**Comment by darlanmendonca** [comment]
@hendriq looks/help me pls

**Comment by darlanmendonca** [comment]
this PR is a fix to https://github.com/darlanmendonca/chai-style/issues/1

**Comment by darlanmendonca** [comment]
@alencar13 checkout pls
